packages/components/src/select-control/test/index.js

```diff
@@ -260,33 +260,27 @@ describe( 'SelectControl', () => {
 			);
 		} );
 
-		describe( 'prop excludeSelectedOptions', () => {
-			it( 'should preserve selected option when focused', async () => {
-				const onChangeMock = jest.fn();
-				const { getByRole } = render(
-					<SelectControl
-						isSearchable
-						showAllOnFocus
-						selected={ options[ 2 ].key }
-						options={ options }
-						onSearch={ () => options }
-						onFilter={ () => options }
-						onChange={ onChangeMock }
-						excludeSelectedOptions={ false }
-					/>
-				);
-				getByRole( 'combobox' ).focus();
-				await waitFor( () =>
-					expect(
-						getByRole( 'option', { name: 'bar' } )
-					).toBeInTheDocument()
-				);
-				// In browser, the <Button> in <List> component is automatically "selected" when <Control> lost focus.
-				// I was not able to produce the same behaviour with unit test, but a click on the currently
-				// selected option should be sufficient to simulate the logic in this test.
-				userEvent.click( getByRole( 'option', { selected: true } ) );
-				expect( onChangeMock ).toHaveBeenCalledTimes( 0 );
-			} );
+		it( 'should not have a selected option on initial render', async () => {
```

Comment:
The control _should_ have a selected option on initial render if supplied a `selected` prop that matches an option's `key` should it not?

Reply:
This test case was the result of my misunderstanding 😅 . I've changed it to `should not automatically select first option on focus`